packages/smooth_app/pubspec.lock

```diff
@@ -148,7 +148,7 @@ packages:
     description:
       path: "packages/camera/camera_platform_interface"
       ref: smooth_camera
-      resolved-ref: d85e1a0df41547cb48c912afc48915dc81a6370a
+      resolved-ref: bc474ef0647d9f865857198e8af000702c72aeb4
```

Comment:
@M123-dev @g123k does that mean we didn't release the scan improvements, since the dependancy was frozen

Reply:
Looks like it

Reply:
But it could also be that we used the newer versions as we call pub upgrade in the release 

https://github.com/openfoodfacts/smooth-app/blob/develop/ci/pub_upgrade.sh